src/components/10-atoms/button/demo.js

```diff
@@ -68,4 +69,14 @@ storiesOf('Components|Atoms/Button/Demos', module)
     const wrapper = document.createElement('div');
     render(template, wrapper);
     return wrapper;
-  });
+  })
+  .add(
+    'Button css width',
+    () =>
+      '<axa-button style="width: 80%">Next step</axa-button><axa-text>You can simply set width by setting css width.</axa-text>'
```

Comment:
Please mention the width that is set (`width: 80%`) so that the user sees what's going on.